setup/fs

```diff
@@ -115,13 +115,21 @@ probe_filesystem()
 	grep -w $fs /proc/filesystems || log_cmd modprobe $fs
 }
 
+is_null_blk()
+{
```

Comment:
any common place we can put this function to avoid duplication?

Reply:
i'm thinking of putting is_mount_on_root and is_null_blk in lib/fs_ext.sh to avoid duplication for those 2 functions, but i have to add '. $LKP_SRC/lib/fs_ext.sh' to top of fio-setup-basic, will that be fine?